source/lac/petsc_precondition.cc

```diff
@@ -488,32 +489,32 @@ namespace PETScWrappers
     AssertThrow (ierr == 0, ExcPETScError(ierr));
 
     if (additional_data.output_details)
-      PetscOptionsSetValue("-pc_hypre_boomeramg_print_statistics","1");
+      {
```

Comment:
I always add brackets out of habbit, even if I don't need them. It is an old habbit..